auction9-frontend/src/app/services/auction.service.ts

```diff
@@ -91,4 +91,16 @@ export class AuctionService {
    realizeAuctionById(auction, status) {
      return this.http.put(`${environment.baseUrl}/myauctions/${auction.auctionID}`, { auction, changeStatus: status }).toPromise();
    }
+
+  /* postNewBid - create new bid for selected auction
+   * Method: POST
+   * Path: /auctions/id/new
+   */
+   createNewBid(auction, newBid) {
+     return this.http.post(`${environment.baseUrl}/auctions/${auction.auctionID}/new`,
+       {
+         auction,
```

Comment:
seems unnecessary?

Reply:
remove auction from body as we will use auctionid from path parameters